Sming/SmingCore/Network/MqttClient.h

```diff
@@ -316,9 +322,9 @@ class MqttClient : protected TcpClient
 
 #ifndef MQTT_NO_COMPAT
 	// @deprecated
-	MqttMessageDeliveredCallback onDelivery = 0;
+	MqttMessageDeliveredCallback onDelivery = nullptr;
 	// @deprecated
-	MqttStringSubscriptionCallback subscriptionCallback = 0;
+	MqttStringSubscriptionCallback subscriptionCallback = nullptr;
```

Comment:
Using nullptr instead of 0 for old-style C++ delegates might not work as expected. Has to be checked.